### PR TITLE
update-api-args

### PR DESCRIPTION
**Pull request by calvix**
towards https://github.com/giantswarm/giantswarm/issues/4211

### CODE REVIEW

File: templates/master.yaml.tmpl [hunk 1]
```diff
@@ -2206,7 +2204,7 @@ storage:
                 - --bind-address=$(HOST_IP)
                 - --etcd-prefix=giantswarm.io
                 - --authorization-mode=RBAC
-                - --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,PodSecurityPolicy,PersistentVolumeClaimResize,DefaultStorageClass,Priority
+                - --enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,PodSecurityPolicy,PersistentVolumeClaimResize,DefaultStorageClass,Priority,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook
```
Comment (r7vme) [inline templates/master.yaml.tmpl]: Do you know why we enable those 3 admission plugins?

Comment (calvix) [reply templates/master.yaml.tmpl]: https://github.com/giantswarm/giantswarm/issues/4211#issuecomment-422751994
@pipo02mix wanted them so i assume he has reasons

Comment (r7vme) [reply templates/master.yaml.tmpl]: ack

Comment (pipo02mix) [reply templates/master.yaml.tmpl]: I have arrived late. Yes we want to offer the possibility to configure admission webhooks (validation/mutation) to customers. For version 1.10 and newer they are recommended to be configured as default https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers#is-there-a-recommended-set-of-admission-controllers-to-use

**Pull request merged**